### PR TITLE
Nuri/update: Updated market page for stocks and indices

### PR DESCRIPTION
**Pull request by njazuli-deriv**
Changes:

-   Update stocks and indices sections for market page

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [X] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — CHANGES_REQUESTED** [review]
Hi @njazuli-deriv , from the design the hero graphic is different. Also you may need to replace _images/common/deriv-platform-banner.png_ with phone image found in the design.

**Comment by njazuli-deriv** [comment]
> Hi @njazuli-deriv , from the design the hero graphic is different. Also you may need to replace _images/common/deriv-platform-banner.png_ with phone image found in the design.

Hi @arome-binary , for footer banner, there will be no changes for this card yet since we are focusing more on `Stocks & Indices` details first.

File: src/pages/markets/static/style/_markets-style.js [hunk 1]
```diff
@@ -6,6 +6,13 @@ import device from 'themes/device'
 export const Col = styled(Flex)`
     max-width: 129px;
     padding: 0 4px;
+
+    @media ${device.tabletL} {
+        max-width: ${({ fullwidth }) => (fullwidth ? '80px' : '129px')};
```
Comment (yashim-deriv) [inline src/pages/markets/static/style/_markets-style.js:11]: need to be in snake_case

File: src/pages/markets/components/helper/_available-trades.js [hunk 2]
```diff
@@ -114,7 +119,7 @@ const CardContainer = styled(Flex)`
         }}
     }
     @media ${device.tabletL} {
-        width: 100%;
+        /* width: 100%; */
```
Comment (yashim-deriv) [inline src/pages/markets/components/helper/_available-trades.js:122]:
```suggestion
```
This probably can be removed

File: src/pages/markets/static/content/_margin.js [hunk 3]
```diff
@@ -11,18 +11,24 @@ import {
...
     SmartFXDetails,
     SmartGoldIndexDetails,
     StepIndicesDetails,
     VolatilityIndicesDetails,
+    AmericanIndicesDetails,
+    AsianIndicesDetails,
+    EuropeanIndicesDetails,
 } from './_details'
-import { OTCGERMAN } from 'components/elements/symbols'
+// import { OTCGERMAN } from 'components/elements/symbols'
```
Comment (yashim-deriv) [inline src/pages/markets/static/content/_margin.js:31]:
```suggestion
```
can be removed

File: src/pages/markets/static/style/_markets-style.js (hunk 1 above)
Comment (njazuli-deriv) [reply src/pages/markets/static/style/_markets-style.js]: fixed in latest commit

File: src/pages/markets/components/helper/_available-trades.js (hunk 2 above)
Comment (njazuli-deriv) [reply src/pages/markets/components/helper/_available-trades.js]: removed in latest commit

File: src/pages/markets/components/helper/_available-trades.js [hunk 4]
```diff
@@ -100,9 +105,9 @@ const CardContainer = styled(Flex)`
         left: 0;
         z-index: -1;
         border-bottom: none;
-        border-radius: 10px 30px 0 0;
+        border-radius: 8px 8px 0 0;
         background: #f3f3f3;
```
[the remark this answers is not in the document]
Comment (njazuli-deriv) [reply src/pages/markets/components/helper/_available-trades.js]: fixed in latest commit

File: src/pages/markets/components/helper/_available-trades.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (njazuli-deriv) [reply src/pages/markets/components/helper/_available-trades.js]: removed in latest commit

File: src/pages/markets/components/sub-markets/_digital-options.js [hunk 5]
```diff
@@ -18,15 +18,15 @@ const DigitalOptions = ({ market_name, options_list }) => (
     <SectionContainer padding="4rem 0 8rem">
         <ContentWrapper>
             <Descriptions>
-                <StyledText align="center">
+                <StyledText font_size={'16px'} align="center">
```
[the remark this answers is not in the document]
Comment (njazuli-deriv) [reply src/pages/markets/components/sub-markets/_digital-options.js]: fixed in latest commit

File: src/pages/markets/static/content/_margin.js (hunk 3 above)
Comment (njazuli-deriv) [reply src/pages/markets/static/content/_margin.js]: removed in latest commit

**Review by yashim-deriv — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**